client/components/woopay/save-user/checkout-page-save-user.js

```diff
@@ -93,23 +87,9 @@ const CheckoutPageSaveUser = ( { isBlocksCheckout } ) => {
 				data: data,
 			} ).then( () => {
 				setUserDataSent( ! shouldClearData );
-				// Cart returned from `extensionCartUpdate` clears these as these fields are not sent to backend by blocks when added.
-				// Setting them explicitly here to the previous user input.
-				cart.setShippingAddress( {
-					phone: shippingPhone,
-				} );
-				cart.setBillingAddress( {
-					phone: billingPhone,
-				} );
```

Comment:
I noticed that one of the root causes for this issue is that the "billing" phone gets wiped out if the "billing same as shipping" is checked and the WooPay phone is entered (see video).

<details>
<summary>Video recording</summary>

https://github.com/Automattic/woocommerce-payments/assets/273592/2840337d-0ee8-4e37-ac2d-29a10d752405

</details>


But removing these calls still seems to be correctly setting shipping & billing phone to the backend and to Stripe. So, it seems to me that these calls have become redundant and can be removed, since they're now causing issues.